som_generationkwh/report/report_amortization_gkwh.mako

```diff
@@ -1,6 +1,7 @@
 <%
     report = objects[0]
     data = report.investmentAmortization_notificationData()
+    from babel.numbers import format_currency
```

Comment:
Babel seems to be a new package. Add it to the requirements.txt file.

Reply:
https://github.com/Som-Energia/somenergia-generationkwh/pull/38/commits/73edc213b75f9f93cb29501986bdc236cff84e7b